v2/checks/vulnerabilities/check.go

```diff
@@ -10,7 +11,45 @@ import (
 // check is a check that verifies if there's an sbom attached with
 // the container image
 type check struct {
-	sbomClient voucher.SBOMClient
+	sbomClient            voucher.SBOMClient
+	failOnVulnerabilities map[string]bool
+	failOnSeverity        string
+}
+
+type SbomVulnerabilityError struct {
+	VulnerabilitiesFound []string
+}
+
+func (err SbomVulnerabilityError) Error() string {
+	output := fmt.Sprintf("vulnernable to %d vulnerabilities: ", len(err.VulnerabilitiesFound))
+
+	for i, vulnerability := range err.VulnerabilitiesFound {
+		if i != 0 {
+			output += ", "
+		}
+		output += vulnerability
+	}
+	return output
```

Comment:
```suggestion
	return output + strings.Join(err.VulnerabilitiesFound, ", ")
```

Or you could do it within the `Sprintf()`.